components/x-interaction/src/InteractionClass.jsx

```diff
@@ -7,26 +7,32 @@ export class InteractionClass extends Component {
 	constructor(props, ...args) {
 		super(props, ...args);
 
-		this.state = props.initialState;
-		this.state[loading] = false;
+		this.state = {
+			[loading]: false,
```

Comment:
Interesting - why did you choose to use a symbol here? (nothing wrong with it, just intrigued!)

Reply:
this was me, it's not a `Symbol` symbol as that didn't work with preact, but a string. it's so that when inspecting the state of any interaction component it's obviously private/internal